BraveRewardsExample/ViewController.swift

```diff
@@ -106,6 +130,12 @@ class ViewController: UIViewController {
     
     setupRewards()
     braveRewardsPanelButton.setImage(RewardsPanelController.batLogoImage, for: .normal)
+//    DispatchQueue.main.asyncAfter(deadline: .now() + 15) {
```

Comment:
Remove?

Reply:
Ah.. was testing postnotification forgot to remove